hpx-sys/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub mod ffi {
         fn hpx_partial_sort_comp(src: &mut Vec<i32>, last: usize, comp: fn(i32, i32) -> bool);
     }
 }
+pub use self::ffi::*;
```

Comment:
Hello,
Could you please explain this change?

Reply:
for ffi bindings, it's generally not considered good practice to use `pub use self::ffi::*` because you want to maintain a clear separation between the ffi fun and the root of the crate. 
this helps in distinguishing between `unsafe/raw ffi calls` and any safe abstractions built on top of them. hence, there's often a separate crate for ffi bindings(hpx-sys) which is then used by a higher-level, more Rust-idiomatic crate(hpx-rs)

Reply:
@Connor-GH could you please remove this as @pingu-73 has described.

Reply:
Okay, after doing more research, ffi as a namespace seems pretty common, as it is used in glib, gtk, and cairo. My original intention was to give the examples a more "seamless" feel, but it should not be done this way. My next proposal is that more ffi wrappers be written in the hpx_sys namespace so that the ffi functions don't show up in code as often. Example: create_c_args instead takes Args and does the conversion in the library. Alternatively, hpx_sys::ffi::init() could be replaced in user code with an hpx_sys::init that takes simply a function and Args.